examples/nextjs-with-typescript/components/media-chrome-react/stateMgmt.js

```diff
@@ -0,0 +1,74 @@
+import React, { createContext, useContext, useMemo } from 'react';
+// NOTE: This is an official package published by the React team and part of their monorepo: https://github.com/facebook/react/tree/main/packages/use-sync-external-store
+// We may be able to migrate to https://react.dev/reference/react/useSyncExternalStore instead if preferred
+import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/with-selector.js';
+import createMediaStore from 'media-chrome/dist/mediaStore.js';
+
+// The context is the thing that allows for a shared... context that propagates outside of the (V)DOM/Component tree.
+// Contexts have values. Our value will be the the media ui store (aka the "guts" of Media Controller state mgmt)
+export const ReactMediaChromeContext = createContext(null);
```

Comment:
Could we simplify the naming to just MediaContext?

Reply:
Yup!